create_group_trigger.yml

```diff
@@ -13,13 +13,18 @@
       severity: 'HIGH'
       state: 'present'
       group_conditions:
-        trigger_mode: 'FIRING'
-        conditions:
-          - name: 'Test Condition'
-            type: 'THRESHOLD'
-            data_id: 'test_condition'
-            operator: 'GT'
-            threshold: 0.8
+        - name: 'Test Condition 01'
```

Comment:
Excellent, this allowed you to drop the nesting :drum: 
Consider calling it just `conditions:` now.